app/views/docs/realtime.phtml

```diff
@@ -410,11 +410,9 @@ subscription.close()</code></pre>
     "payload": {
         "field1": "lorem ipsum",
         "field2": [],
-        "$read": [
-            "user:627a958ded6424a98a9f"
-        ],
-        "$write": [
-            "user:627a958ded6424a98a9f"
+        "$permissions": [
+            "read(user:627a958ded6424a98a9f)",
+            "read(user:627a958ded6424a98a9f)"
```

Comment:
I think it is `read("user:627a958ded6424a98a9f")`, not `read(user:627a958ded6424a98a9f)`. That would follow query syntax.